### ISSUE TITLE
Change rules for "link it too long"

### ISSUE THREAD

**Issue by tordans** [TheFive/osmbc#924]
As a user, I noticed that weekly osm links are often one word or very short. I find this bad UX and accessibility since the text of a link should describe the target well. My rule of thumb is: If I scan the text and only read links, I should have an idea what the linked page is about.

Until now I though that is something that the editors decide.

However, today I created a text for a submission and received an error message that my link is too long. It looks like the tool guides editors to create short links. 

I want to suggest to change that rule to (a) allow longer links and (b) maybe even consider guiding editor to do the opposite, to add longer links (not as an error, but a notice).

**Comment by fredao** [comment]
Hi Tobias, thanks for the comment.
Here are our rules: https://osmbc.openstreetmap.de/osmww/wiki/Grunds%C3%A4tze-zur-Formulierung-und-Formatierung?hl=verb 

"    Normalerweise verlinken wir auf dem Verb.
    Manchmal ist eine Verlinkung aber auf dem Kern der Aussage sinnvoller: Beispiel:
        Bundesverkehrsminister Dobrindt gab den Fond "mFund" frei.
    Hier wird nicht auf gab verlinkt.
"
und der Hinweis, dass ein Link zu lang sein könnte hat sich bewährt
Es ist im übrigen nur ein Hinweis. ;_)

**Comment by tordans** [comment]
> Es ist im übrigen nur ein Hinweis. ;_)

Thanks. The red text looked like an error to me. Good to know that I could ignore it :)

> Here are our rules: https://osmbc.openstreetmap.de/osmww/wiki/Grunds%C3%A4tze-zur-Formulierung-und-Formatierung?hl=verb

Thanks, good to know it's specified. I case it comes up, maybe review those recommendations and consider updating the current rules.
- https://www.nngroup.com/articles/better-link-labels/
- https://sketchplanations.com/click-here

I think both pages underline my take to choose a more explanatory link that just a verb.
